### PR TITLE
hwdef: update PixFlamingo_f767 README

### PR DESCRIPTION
**Pull request by tsubashchandar**

### CODE REVIEW

**Comment by tsubashchandar** [comment]
@Hwurzburg  can you review this

File: libraries/AP_HAL_ChibiOS/hwdef/PixFlamingo-F767/README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Contact dheeranlabs@gmail.com for sales
 |  5   | I2C2_SDA | +3.3V |
 |  6   |    GND   |  GND  |
 
-**GPS2**
+**SERIAL5**
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/PixFlamingo-F767/README.md:73]: this is a 5 pin, not 6 pin connector as shown on pinout diagram

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

**Review by peterbarker — COMMENTED** [review]
Commit message needs to be fixed - something like:
`hwdef: update PixFlamingo_f767 README`

**Review by Hwurzburg — APPROVED** [review]
squashed the commits for you...commit is now in proper form

**Pull request merged**